src/literature/components/AuthorLink.jsx

```diff
@@ -18,6 +18,16 @@ class AuthorLink extends Component {
     return `//inspirehep.net/search?cc=Institutions&p=institution:"${affiliation}"`;
   }
 
+  getFullName() {
+    const { author } = this.props;
+    if (author.has('first_name')) {
+      const firstName = author.get('first_name');
+      const lastName = author.get('last_name', '');
+      return `${firstName} ${lastName}`;
```

Comment:
what about the third part for `jr.`, etc.?

Reply:
the backend is not ready yet, we'll do it in a separate PR